src/components/Button.js

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ buttonName }) => <button type="button">{buttonName}</button>;
+const Button = ({ buttonName, color, wide }) => (
+  <button type="button" className={`button border ${wide} ${color}`}>
+    {buttonName}
+  </button>
+);
 
 Button.propTypes = {
   buttonName: PropTypes.string,
+  color: PropTypes.string,
+  wide: PropTypes.string,
 };
 
 Button.defaultProps = {
   buttonName: '',
+  color: '',
+  wide: '',
```

Comment:
`wide` should not have a default prop, it should either be `true` or `false`. Let's make it a required prop